Source/Samples/01_HelloWorld/CMakeLists.txt

```diff
@@ -26,6 +26,11 @@ set (TARGET_NAME 01_HelloWorld)
 # Define source files
 define_source_files (EXTRA_H_FILES ${COMMON_SAMPLE_H_FILES})
 
+if (WEB)
+    # Custom emscripten template
+    define_resource_dirs (HTML_SHELL ${CMAKE_SOURCE_DIR}/web/application.html )
+endif()
+
```

Comment:
Any chance to get this thing extracted into common place?
Maybe `setup_main_executable`?
I wonder if @weitjong has good ideas on the subject.

Reply:
Found a better place https://github.com/urho3d/Urho3D/blob/master/CMake/Modules/UrhoCommon.cmake#L1109-L1115